### PR TITLE
[WIP] TE-8522 Fix up missing GA moves

### PR DESCRIPTION
**Pull request by dmytro-dymarchuk**
Ticket: https://spryker.atlassian.net/browse/TE-8522

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,6 +6,20 @@ on:
         branches:
             - master
     workflow_dispatch:
+    schedule:
+        - cron: "0 0 * * *"
```
Comment (dereuromark) [inline .github/workflows/ci.yml:10]: Lets drop schedule

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -5,7 +5,22 @@ on:
     push:
         branches:
             - master
-    workflow_dispatch:
```
Comment (dereuromark) [inline .github/workflows/ci.yml:8]: This needs to stay

Comment (dmytro-dymarchuk) [reply .github/workflows/ci.yml:8]: Ok. I'm working on it right now, so I will let you know when work is finished and you can proceed review.

Comment (dereuromark) [reply .github/workflows/ci.yml:8]: I am just outlining things that shouldnt have been touched, and you can include them during that process already.

**Comment by dereuromark** [comment]
Did you run this locally first?
Doing so will most likely be faster to get a working prototype, and then you can still push it to CI.

**Comment by dmytro-dymarchuk** [comment]
> Did you run this locally first?
> Doing so will most likely be faster to get a working prototype, and then you can still push it to CI.

No, I'm not sure this way is faster because I have to clean the local environment. And I suppose I can have a problem with ga environment but all is good locally at the same time.  But I can try.

**Comment by dereuromark** [comment]
That looks great! Nice work - can we get the build green (regarding the CS errors etc) or do we need to do something here?

**Comment by dmytro-dymarchuk** [comment]
> That looks great! Nice work - can we get the build green (regarding the CS errors etc) or do we need to do something here?

I have several things to work on.
1. fix cs.
2. Resolve issues with --prefer-lowest.
3. Fix problem with exit code because build with --prefer-lowest has error but status is green. 
4. Check how eco-ci will work for repositories that don't contain any test. E.g https://github.com/spryker-eco/loggly

I'm working on it right now.

**Comment by dereuromark** [comment]
Yeah, repos without any tests should be disallowed :)

**Comment by dmytro-dymarchuk** [comment]
Is duplicate https://github.com/spryker-eco/payone/pull/72

**Pull request closed without merging**